.github/workflows/ci.yml

```
@@ -7,39 +7,59 @@ on:
+          MYSQL_DATABASE: taste_ci_test
+          MYSQL_USER: testuser
+          MYSQL_PASSWORD: test1234
+        ports:
+          - 3306:3306
+        options: >-
+          --health-cmd="mysqladmin ping -h 127.0.0.1 -uroot -proot1234"
+          --health-interval=5s
+          --health-retries=20
+
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)